addon-test-support/@ember/test-helpers/dom/-target.ts

```diff
@@ -0,0 +1,3 @@
+type Target = string | Element;
```

Comment:
I think there is actually an `EventTarget` type, is that better here?

Reply:
`Target` in this case means "what you can pass into e.g. `click()`", not the `target` property of an `Event`